src/dlgTriggerEditor.cpp

```diff
@@ -7047,6 +7057,21 @@ void dlgTriggerEditor::slot_grab_key()
     }
 }
 
+#ifdef QT_GAMEPAD_LIB
+
+void dlgTriggerEditor::slot_grab_button()
+{
+    if( ! mpHost ) return;
+
+    dlgButtonSelect * pDlg = new dlgButtonSelect( this );
```

Comment:
The dialog needs a title and layout fixes - I believe you're coming up with a better design already.